doc/reference/jwt.rst

```diff
@@ -26,3 +26,33 @@ by this issuer. This is how NAV maps an incoming token to the correct configurat
 Likewise ``aud`` must match the ``aud`` claim of the tokens. This is a security measure to make sure a NAV instance
 only accepts tokens meant for it. Otherwise the NAV instance would accept all JWT tokens generated by an issuer.
 In other words, anyone using the same token issuer could generate tokens and gain access to your NAV instance.
+
+.. _local-jwt-configuration:
+Configuring local JWT token generation
+--------------------------------------
+Local JWT token generation is configured in the ``nav`` section of :file:`jwt.conf`.
+
+You must configure three things: a private key, a public key, and the issuer name.
+The keys must be a private/public RSA keypair in PEM format.
+The issuer name is used for the ``iss`` and ``aud`` claims of generated tokens.
+The issuer name can be whatever you want, but a natural choice is the domain name of your NAV instance (e.g. ``nav.example.com``).
+An example configuration is shown below::
+
+  [nav]
+  # Absolute path to private key in PEM format
+  private_key=/path/to/jwtRS256.key
+  # Absolute path to public key in PEM format.
+  public_key=/path/to/jwtRS256.key.pub
+  # Used for the 'iss' and 'aud' claims of generated tokens.
+  name=nav
+
+The keypair can be generated like so::
+
+  ssh-keygen -t rsa -b 4096 -m PEM -f jwtRS256.key
+  # Don't add passphrase
+  openssl rsa -in jwtRS256.key -pubout -outform PEM -out jwtRS256.key.pub
```

Comment:
How come openssl cannot also be use to generate the key?

Reply:
Might be possible, this is how I've been doing it and it has worked well. I'll see if  I can find the openssl equivalent just to have fewer dependencies here

Reply:
`openssl genrsa -out testopenssl.pem 4096` seems to be the equivalent